### PR TITLE
Add new Matlab Online workflow

### PR DESCRIPTION
**Pull request by pwalczysko**
cc @jburel 

Suggesting an addition as discussed.

### CODE REVIEW

File: docs/matlabonline.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Analyze OMERO data using MATLAB Online
+======================================
+
+This workflow was contributed by community, see https://github.com/mathworks/Open-Microscopy-Data-MATLAB. It allows to work with OMERO data via MATLAB Online. In this way, there is no need to download and install MatLab locally as MatLab is running in your browser. You will need a valid Mathworks account. An easy to understand environment with Jupyter Notebooks is provided. Follow the steps on the https://uk.mathworks.com/matlabcentral/fileexchange/156034-open-microscopy-data-matlab.
```
Comment (jburel) [inline docs/matlabonline.rst]: it should be MATLAB

Comment (pwalczysko) [reply docs/matlabonline.rst]: fixed in https://github.com/ome/omero-guide-matlab/pull/5/commits/594756cda921c29c8dcc12846660480586db1353

File: docs/matlab.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Analyze OMERO data using MATLAB
 ===============================
 
-Matlab is a powerful programming platform. We show here you can analyze data stored in OMERO using Matlab.
+MATLAB is a powerful programming platform. We show here you can analyze data stored in OMERO using MATLAB.
```
Comment (jburel) [inline docs/matlab.rst]:
```suggestion
MATLAB is a powerful programming platform. We show here how you can analyze data stored in OMERO using MATLAB.
```

File: docs/matlabonline.rst [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Analyze OMERO data using MATLAB Online
+======================================
+
+This workflow was contributed by community, see https://github.com/mathworks/Open-Microscopy-Data-MATLAB. It allows to work with OMERO data via MATLAB Online.
```
Comment (jburel) [inline docs/matlabonline.rst]:
```suggestion
This workflow was contributed by the community, see https://github.com/mathworks/Open-Microscopy-Data-MATLAB. It allows to work with OMERO data using MATLAB Online. 
```

File: docs/matlabonline.rst [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Analyze OMERO data using MATLAB Online
+======================================
+
+This workflow was contributed by community, see https://github.com/mathworks/Open-Microscopy-Data-MATLAB. It allows to work with OMERO data via MATLAB Online. 
+
+In this way, there is no need to download and install MATLAB locally as MATLAB is running in your browser.
```
Comment (jburel) [inline docs/matlabonline.rst]:
```suggestion
This means that users do no need to download and install MATLAB locally as MATLAB is running in your browser.
```

File: docs/matlabonline.rst [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+
+You will need a valid MathWorks account. An easy to understand environment with Jupyter Notebooks is provided.
```
Comment (jburel) [inline docs/matlabonline.rst]:
```suggestion
You will just need a valid MathWorks account. An easy to understand environment with Jupyter Notebooks is provided.
```

**Comment by pwalczysko** [comment]
Should we merge as discussed ? (I do not have the merge button here)

**Pull request merged**